### DISCUSSION TITLE
Add option to block only promotions

### DISCUSSION THREAD

**Discussion by BobbyByrne** [simple-login/app#1844] [Feature Requests]
Feature: Provide a toggle that extends the existing "Unsubscribe" functionality. This toggle would have an additional option to block promotional emails and allow all others. Additionally, having an option in preferences to set all new aliases to Block Promotions by default would be a big help.

So three options would be available

- Allow All
- Block Promotions
- Block All

Frequently while shopping online you'd like to receive purchase receipts, shipping notifications, and account change emails from an online store, but have promotional content blocked.

This feature is available in Firefox Relay and it's quite effective. Below is an example:
 
![image](https://github.com/simple-login/app/assets/7388850/257a2d31-9fde-4ee2-9b47-a2b2673250b7)

This missing feature is the biggest reason why I'm still on Firefox Relay despite paying for Proton premium which includes SimpleLogin.

**Comment by jshargrave** [comment]
Yes, I would really like this feature. Or at least some blocking feature using wild card text parsing in the email subject/body would be much appreciated.
